.github/workflows/c3-dependabot-versioning-prs.yml

```diff
@@ -19,10 +19,10 @@ jobs:
       - uses: pnpm/action-setup@v2
         with:
           version: 8.8.0
-      - name: Use Node.js 16.18
+      - name: Use Node.js 18
         uses: actions/setup-node@v3
         with:
-          node-version: 16.18
+          node-version: 18.20.2
```

Comment:
can we pin this to latest v18.x.x rather than the specific v18.20.2?

Reply:
I'm not keen on doing that because that means that running a build at two different times might have different results. I am also not worried about keeping on the leading edge given that we have spent the last year stuck on v16 😄

Reply:
In the pnpm PR that will follow this, I move all the dependency installations into a single common action so we only have to change the version in one place.